web/packages/api/src/parachains/assethub.ts

```diff
@@ -0,0 +1,244 @@
+import { AssetMap, PNAMap } from "src/assets_v2"
```

Comment:
Why not use the relative path `../assets_v2`?

Reply:
will fix